docs/_news/news13.md

```diff
@@ -0,0 +1,70 @@
+- [A warning appears if a project directory is added to multiple connections](https://github.com/eclipse/codewind/issues/2922). Adding a project directory to multiple connections is still allowed.
+- If you disable remote connections, they [remain disabled after you restart VS Code](https://github.com/eclipse/codewind/issues/2823).
+- The location of the Codewind extension logs is [reported at the beginning of the Codewind output stream](https://github.com/eclipse/codewind-vscode/pull/600).
+- The Codewind output stream [logs timestamps and time elapsed for each command](https://github.com/eclipse/codewind-vscode/pull/607/commits/bb55fe7f2c5ea5fc5e92962f5a83ce960d768e0e).
+
+#### List of Fixes
+- In Codewind for Eclipse and VS Code:
+  - When you use the IDE to upgrade Codewind, the IDE [removes the old version of the Codewind CLI](https://github.com/eclipse/codewind/issues/2869) from the disk to save space.
+- In Codewind for Eclipse:
+  - [A progress monitor appears when certain actions are selected](https://github.com/eclipse/codewind/issues/2997). For example, a progress monitor appears while Manage Template Sources retrieves data from a connection.
+  - The local connection [no longer shows a disconnected error when Eclipse is restarted](https://github.com/eclipse/codewind/issues/2963).
+  - The [dialog is improved](https://github.com/eclipse/codewind/issues/2924) in the window that appears if you try to add a project that is already deployed on another connection.
+  - The `Widget is disposed` exception [no longer appears](https://github.com/eclipse/codewind/issues/2917) when you update the project overview page.
+  - Codewind remembers the state of remote connections, whether connected or disconnected, [when you restart Eclipse](https://github.com/eclipse/codewind/issues/2823).
+  - The refresh action on the root element in the Codewind Explorer view [checks for new connections](https://github.com/eclipse/codewind/issues/2832).
+  - **Restore defaults** on the Codewind preferences page [resets the fields to their defaults](https://github.com/eclipse/codewind/issues/2836).
+- In Codewind for VS Code:
+  - [Project files can be deleted on Windows](https://github.com/eclipse/codewind/issues/2456). Also, a warning message appears if the extension is going to reload because the workspace folders are changed.
+  - [Some of the colors in the webviews are updated because they changed in VS Code 1.45](https://github.com/eclipse/codewind-vscode/pull/596). Also, the table header in the Template Sources and Image Registry pages is fixed.
+  - [Cancel a connection that connects after restarting VS Code](https://github.com/eclipse/codewind/issues/2824).
+  - [Webviews close when the extension is deactivated](https://github.com/eclipse/codewind/issues/2878).
+- The Performance Dashboard includes new capabilities paneling.
+- Run load concurrently against multiple projects.
+- Diagnostics collection is included with `cwctl`.
```

Comment:
we're not currently calling out the use of cwctl externally so this should probably come out